File: live/src/main/java/com/zky/live/fragment/TestFragement.java
package com.zky.live.fragment;

import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

/**
 * Created by lk
 * Date 2019-11-08
 * Time 14:45
 * Detail:
 */
public class TestFragement extends Fragment {


    public static TestFragement getInstance(){
        return new TestFragement();
    }

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }
}

File: live/src/main/java/com/zky/live/fragment/TestFragement.kt
package com.zky.live.fragment

import android.os.Bundle
import androidx.fragment.app.Fragment

/**
 * Created by lk
 * Date 2019-11-08
 * Time 14:45
 * Detail:
 */
class TestFragement : Fragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    companion object {
        val instance: TestFragement
            get() = TestFragement()
    }
}